backend/src/middleware/validation/index.js

```diff
@@ -22,10 +22,27 @@ const validateUrl = async (resolve, root, args, context, info) => {
   }
 }
 
+const validateComment = async (resolve, root, args, context, info) => {
+  const COMMENT_MIN_LENGTH = 1
+  const content = args.content.replace(/<(?:.|\n)*?>/gm, '').trim()
+  if (!args.content || content.length < COMMENT_MIN_LENGTH) {
+    throw new UserInputError(`Comment must be at least ${COMMENT_MIN_LENGTH} character long!`)
+  }
+  const NO_POST_ERR_MESSAGE = 'Comment cannot be created without a post!'
+  const { postId } = args
+  if (!postId) {
+    throw new UserInputError(NO_POST_ERR_MESSAGE)
+  }
```

Comment:
why do we need the `postId` to update a comment?
I think this would be a good test case, cause I think you could write a test that exposes that a `postId` can be passed in that is not associated with an actual post in the database.

This is why in the `CreateComment` resolver, there is a query to see that the post is in the database, otherwise does not create a comment. we don't want orphaned comments that either don't have an author or a post associated with them.

For update comments, I think we can assume that the comment was created with a post, no?
Is there something we are doing with this `postId` that I'm missing?

Ahhh, I think it is just to refactor the `CreateComment` resolver, since @roschaefer asked to make the validations in `validation/index.js`?

If so, I would put it back and not worry about that in this `PR` as it's not related to updating a comment, and we shouldn't be requiring a `postId` to be passed in, as it's not relevant to updating a comment.

We can open a separate ticket to deal with refactoring the `CreateComment` validations